### ISSUE TITLE
JAWS cannot read/see math content inside a fieldset legend element

### ISSUE THREAD

**Issue by LauraOH** [FreedomScientific/standards-support#917]
### Summary
In Chrome, when MathML is present in a legend element, JAWS does not read the math content as a part of the legend for the group when tabbing through the form, nor can it read it when reading the content.  Thus the MathML content when appearing as part of a legend is completely inaccessible to JAWS.

### Steps to reproduce

1. Go to https://codepen.io/LauraFO/details/LEpeZyM
2. Set focus in the text field
3. Tab to the radio button in the fieldset
4. Observe when reading the group label, JAWS only reads the text "Expand the following equation" and does not read the math content.

1. Go to https://codepen.io/LauraFO/details/LEpeZyM
2. Set focus in the text field
3. Switch to Virtual PC Cursor.
4. Press down arrow to read the fieldset legend.
5. Observe when reading the group label, JAWS only reads the text "Expand the following equation" and does not read the math content.

### Expected Result

JAWS would read out Math content as a part of the legend.

### Jaws version and build number
2024.2411.16
2025.2508.120

### Related issue: 
#526  JAWS not reading MathML present in HTML form fields
In issue #526, there was a workaround to read math content in the label. In this case, there is no workaround available to read the math content in the legend.

### Browser and version
Chrome 139.0.7258
Chrome 140.0.7339.81

**Comment by JeremiahRogersVispero** [comment]
Works as expected in JAWS 2026.2608.25.
